### PR TITLE
Feature/ios/migen 1825 getting started

### PR DESCRIPTION
**Pull request by Daniel-Nielsen**

### CODE REVIEW

**Review by marksdk — APPROVED** [review]

File: src/ios/v3/getting-started/directions.md [hunk 1]
```diff
@@ -14,9 +14,43 @@ eleventyNavigation:
...
 
-* **TODO: How to swap between travel modes**
+Set **travel mode** on your request using the `travelMode` property on `MPDirectionsQuery`:
+
+```swift
+let directionsQuery = MPDirectionsQuery.init(originPoint: origin!, destination: destination!)
+directionsQuery.travelMode = .driving
+```
+
+The travel modes generally applies for outdoor navigation. Indoor navigation calculations are based on **walking** travel mode.
```
Comment (simonlaustsen) [inline src/ios/v3/getting-started/directions.md:56]: This part seems generic, would it make sense to move this part to the shared snippet?
_shared/getting-started/directions/travel-mode.md"_

File: src/shared/getting-started/directions/travel-mode.md [hunk 2]
```diff
@@ -1 +1,3 @@
 ## Change transportation mode
```
Comment (simonlaustsen) [inline src/shared/getting-started/directions/travel-mode.md:1]: I don't see this included anywhere?

**Review by simonlaustsen — COMMENTED** [review]
Two comments :D

File: src/shared/getting-started/directions/travel-mode.md (hunk 2 above)
Comment (Daniel-Nielsen) [reply src/shared/getting-started/directions/travel-mode.md:1]: I do see it in line 47 of src/ios/v3/getting-started/directions.md

File: src/ios/v3/getting-started/directions.md (hunk 1 above)
Comment (Daniel-Nielsen) [reply src/ios/v3/getting-started/directions.md:56]: Yes, I will try adding it to src/shared/getting-started/directions/travel-mode.md

File: src/shared/getting-started/directions/travel-mode.md (hunk 2 above)
Comment (simonlaustsen) [reply src/shared/getting-started/directions/travel-mode.md:1]: Sorry, my bad  👍  I thought it was a new file

File: src/ios/v3/getting-started/directions.md (hunk 1 above)
Comment (simonlaustsen) [reply src/ios/v3/getting-started/directions.md]: 👍

**Review by simonlaustsen — APPROVED** [review]
I find it a bit difficult to review the platform-specific steps and snippets but let's merge! 👍

**Pull request merged**